tests/unit/nupic/utils_test.py

```diff
@@ -84,23 +84,19 @@ def testMovingAverageInstance(self):
     newAverage = ma.next(3)
     self.assertEqual(newAverage, 3.0)
     self.assertListEqual(ma.getSlidingWindow(), [3.0])
-    self.assertEqual(ma.total, 3.0)
```

Comment:
the new implementation of MOvingAverage I took from metrics.py does not have the internal `total` field.